internal/util/validationutil/util_test.go

```diff
@@ -29,6 +30,9 @@ import (
 )
 
 func TestValidateAsNonNegativeInt32(t *testing.T) {
+	if runtime.GOARCH == "386" {
```

Comment:
I think we will need to check against more 32bit architectures, the most important one being `arm`. Here is a gist which lists all `GOARCH` values. I think it makes sense to check against "out of the box" 32bit archs listed here: https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63
```
"386", "arm", "mips", "mipsle"
```
How about adding a new function, something like `it.Is32BitArch() bool` and use it instead of `runtime.GOARCH == "386"`?

Reply:
Great idea, however because of circular dependency I had to move it to a new package it/runtime.